### ISSUE TITLE
View configuration loss possible

### ISSUE THREAD

**Issue by zackkatz** [gravityview/GravityView#1052]
Over the years, we have had customers report every so often that their View Configuration was being wiped out on save. 

A customer has provided the first potential area to reproduce, although we still have not been able to. 

See below: 

```
[BUG report] View configuration loss while refreshing page with view content and updating view setting at the same time

Hi team,
It's been so long since the last contact, this time I'm re-submitting a possible glitch that only occurs in certain combination of actions, which are:
1. Go to the view settting page, and click 'update' button to update the view settings.
2. In the mean time, open a page with that view embedded, and refresh it as well.

I'm doing this because I want to see the result of the updated view content faster, however this caused me an entire day to recover my host since when I take these 2 actions at the same time, I found the view content completely lost...

I don't think it's an environment specific case because it has nothing to do with other plugins. However I got no way to prove it, nor I can provide any form export that can 100% duplicate this issue, it only happens because there's a overlap between the 2 loading pages. If you test it on local server, it might be too fast to occur, even in my host it's not always happening.

```

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/995529792029955/995552682531200)

**Comment by zackkatz** [comment]
HS#14450

**Comment by rafaehlers** [comment]
FYI I tried this several times, I wasn't able to reproduce using this method.

**Comment by soulseekah** [comment]
This race-condition could be related to cache actually, can we get more information of whether they're running a Redis/Memcached-backed object cache or similar? Maybe caching plugins?

**Comment by soulseekah** [comment]
Fixed in #1350 (duplicate)
